File: src/test/java/io/github/bot/terminal/application/domains/TestNitriteDataBaseConfiguration.java
package io.github.bot.terminal.application.domains;

import org.dizitart.no2.Nitrite;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;


@Configuration
public class TestNitriteDataBaseConfiguration {

    @Bean(destroyMethod = "close")
    public Nitrite getNitriteInMemory(){
        return Nitrite.builder()
                .compressed()
                .openOrCreate();
    }

}

File: src/test/kotlin/io/github/bot/terminal/application/domains/TestNitriteDataBaseConfiguration.kt
package io.github.bot.terminal.application.domains

import org.dizitart.no2.Nitrite
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class TestNitriteDataBaseConfiguration {

    @Bean(destroyMethod = "close")
    open fun getNitriteInMemory(): Nitrite = Nitrite.builder()
            .compressed()
            .openOrCreate()

}
